src/Compilers/VisualBasic/Portable/Errors/Errors.vb

```diff
@@ -1957,7 +1957,7 @@ Namespace Microsoft.CodeAnalysis.VisualBasic
         WRN_Experimental = 42380
 
         ' // AVAILABLE                             42381 - 49998
-        ERRWRN_Last = WRN_Experimental
+        ERRWRN_NextAvailable = 42381
```

Comment:
`ERRID.ERRWRN_NextAvailable` is only used in one location, `VisualBasicCompilerDiagnosticAnalyzer.GetSupportedErrorCodes()`, to determine whether each id is within the range of errors and warnings. I agree, we could set the value to a much larger value to avoid updating each time an error or warning is added at the end. That larger value would probably require explanation though - perhaps comments at both the definition and use - so I'll leave as is for now. Any additional errors or warnings will require changing the existing `AVAILABLE` comment range above `ERRWRN_NextAvailable` regardless.